File: 프로그래머스/unrated/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    answer = 1
    dotx1 = dots[0][0]
    doty1 = dots[0][1]
    dotx2 = 0
    doty2 = 0
    for do in dots:
        if do[0] != dotx1:
            dotx2 = do[0]
        if do[1] != doty1:
            doty2 = do[1]
    answer *= (dotx1-dotx2)
    answer *= (doty1-doty2)
    if answer < 0:
        answer *= -1
    return answer

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.java
class Solution {
    public int solution(int[][] dots) {
        int answer = 0;
        int x1 = dots[0][0],x2 = 0,y1 = dots[0][1],y2 = 0;
        
        for(int i=1;i<4;i++){
            if(dots[i][0] != x1 && dots[i][0] != x2)
                x2 = dots[i][0];
            if(dots[i][1] != y1 && dots[i][1] != y2){
                y2 = dots[i][1];
            }
        }
        
        answer = (x1-x2) * (y1-y2);
        if(answer < 0)
            answer *= -1;
        
        return answer;
    }
}